website/docs/reference/resource-configs/target_schema.md

```diff
@@ -4,7 +4,7 @@ description: "Target_schema - Read this in-depth guide to learn about configurat
 datatype: string
 ---
 
-:::note
+:::info
 
 For [versionless](/docs/dbt-versions/core-upgrade/upgrading-to-v1.8#versionless) dbt Cloud accounts and dbt Core v1.9+, this functionality is no longer required. Use the [schema](/reference/resource-configs/schema) config as an alternative to define a custom schema while still respecting the `generate_schema_name` macro. 
 
```

Comment:
there's also a default section below that says 
<img width="453" alt="Screenshot 2024-09-24 at 3 41 26 PM" src="https://github.com/user-attachments/assets/8ed3e61d-0802-4fe7-8afc-8b8a65ae9cfc">

Can we update that for versions 1.9+ / versionless - this is no longer a required config